README.md

```diff
@@ -129,6 +129,22 @@ Then:
 sh pages_builder/push_to_github_pages.sh
 ```
 
+## Development
+
+A local checkout of the frontend-toolkit repo can be shared with locally-running services (i.e. frontend applications)
+as follows, assuming you have a system-wide install of yarn available:
+
+- from this repo, run `yarn link`
+- from each app, run `yarn link digitalmarketplace-frontend-toolkit`
+
+Your frontend apps will then be using your local copy of the frontend-toolkit patterns rather than the version specified
+in their `package.json` - for example, whenever you:
+
+ - rebuild the app's `toolkit` directory by running `make frontend_build`; or
```

Comment:
Oh, this is out-of-date (and in the frameworks repo too), it should have been `make frontend-build` with a dash. It changed some time last year.